compiler/tests-ocaml/basic-modules/recursive_module_init.ml

```diff
@@ -66,3 +66,25 @@ end
 let () =
   List.iter (fun x -> x#go)
     [new Foo.cls; Foo.M.foo(); Lazy.force Foo.M.bar]
+
+module rec Even : sig val is: int -> bool end =
```

Comment:
I try to keep the directory `tests-ocaml` as close as possible to the `testsuite/tests` directory from ocaml repo.

Can you move the test in `compiler/tests-jsoo/test_rec_mod.ml` ?

I'll try to document that the purpose of the various test directories